src/Compilers/Core/Portable/Text/SourceText.cs

```diff
@@ -1012,7 +1020,7 @@ private int[] ParseLineStarts()
 
             return lineStarts.ToArrayAndFree();
         }
-        #endregion
```

Comment:
Damn, I didn't touch these lines... Isn't this set wrong in editorconfig somewhere?